### DISCUSSION TITLE
Improve UX of selecting articles (or other items) in Menuitem

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#47155] [2. Cold Features]
_my obligatory disclaimer for this weeks issues :-)_ 
I am currently teaching a class of kids in working with Joomla. So here and there always the same mistakes happen where we can learn from. If a new user comes into such situations the frustration can be high. But we can help them not to make such mistakes.

### What needs to be fixed
When the modal opens to select an article in the menuitem dialog some kids did not know what to do. clicking on the title is actually not very obvious because normally you would enter the article then. So it would be good to have an "insert button" in front of the item. 

### Why this should be fixed
To improve usability for the newbies :-)

### How would you fix it
an insert Button in front of the item

### Side Effects expected
no

**Comment by brianteeman** [comment]
Not something I see a need for myself but it would be very easy for you to create a pull request to achieve (for j4 only of course)

**Comment by coolcat-creations** [comment]
I could not change a button class in 30+ hours of work for the backend - I doubt I can do something for the Backend unfortunatelly :-(  But I think raising UX issues for Newbies is still very important and the issue is still relevant for me.

**Comment by brianteeman** [comment]
Is this what you want?

![image](https://user-images.githubusercontent.com/1296369/187070729-cc5c613d-3cf9-4d9f-8fc9-4a1551fa63f3.png)

**Comment by coolcat-creations** [comment]
I meant the Popup before, it does not explain what it does...

![grafik](https://user-images.githubusercontent.com/828371/187072811-e250f03f-9fde-4a46-9952-fec942c5a7ef.png)
![grafik](https://user-images.githubusercontent.com/828371/187072889-9a8964dc-dfc4-4be2-bc13-04af857f8912.png)

**Comment by brianteeman** [comment]
ah ok

Not sure I agree with the word Connect but I get the point and will make something later - very easy to do

**Comment by coolcat-creations** [comment]
Thank you! Very curious about the steps to make I look forward to learn

**Comment by chmst** [comment]
I see what elisa means and agree that it is not obvious what to do.
But personally I would prefer someting more decent than extra buttons. Maybe simply styling the link themselves as buttons like in readmore buttons. 
Extra buttons increse the number of elements on the site (a11y)

**Comment by brianteeman** [comment]
I wasnt thinking of buttons. In my screenshot (admittedly the wrong screen) its just a badge.

Anyway its easy to do when its decided. 

But please update the title and label of this issue. It will make @zero-24 life easier
